src/store/actions/index.js

```diff
@@ -1,8 +1,28 @@
-const ADD_TODO = 'ADD_TODO';
+export const ADD_TODO = 'ADD_TODO';
+export const REMOVE_TODO = 'REMOVE_TODO';
+export const EDIT_TODO = 'EDIT_TODO';
 
-function addTodo(todo) {
+
+export function addTodo(todo) {
+    console.log(todo);
     return {
         type: ADD_TODO,
-        payload:todo,
+        payload: todo,        
+    };
+}
+
+
+export function removeTodo(index) {
+    return {
+        type: REMOVE_TODO,
+        item: index,        
+    };
+}
+
+export function editTodo(index, value) {
+    return {
+        type: EDIT_TODO,
+        item: index,     
+        payload: value,   
```

Comment:
you action should contain only type and payload if you need to pass anything other pass it inside of payload as payload{item,value}